applications/tari_base_node/src/grpc/base_node_grpc_server.rs

```diff
@@ -104,8 +105,21 @@ impl BaseNodeGrpcServer {
             software_updater: ctx.software_updater(),
             comms: ctx.base_node_comms().clone(),
             liveness: ctx.liveness(),
+            obscure_grpc_error: ctx.get_obscure_grpc_error(),
         }
     }
+
+    pub fn obscure_error_flag(&self) -> bool {
+        self.obscure_grpc_error
+    }
+}
+
+pub fn obscure_error(obscure: bool, status: Status) -> Status {
+    if obscure {
+        Status::unknown("Error has occurred. Details are obscured.")
```

Comment:
The status code (int) should still be sent, just not the message

Reply:
Done.